src/lib/crypto/krb/prng_os.c

```diff
@@ -91,3 +91,11 @@ krb5_c_random_make_octets(krb5_context context, krb5_data *outdata)
     }
     return 0;
 }
+
+krb5_error_code KRB5_CALLCONV
+krb5_c_random_os_entropy(krb5_context context, int strong, int *success)
+{
+    /* The OS RNGs that accept input from userspace do not increase the
+     * entropy estimate based on this new entropy. */
```

Comment:
In prng.c, krb5_c_random_os_entropy() calls add_entropy_from_device(), which first gets entropy from device in a call to read_entropy_from_device() and then passes it to krb5_c_random_add_entropy() which, in the prng_os.c version, writes it into urandom.